src/controllers/user.controllers.js

```diff
@@ -14,10 +15,11 @@ export const editContact = async (req, res) => {
         },
       },
       {
-        arrayFilters: [{ 'contact._id': req.params._id }],
+        arrayFilters: [{ 'contact._id': editedContactId }],
+        new: true,
       },
     );
-    res.status(201).json({ message: 'Successfully edited contact' });
+    res.status(201).json(user.contacts);
```

Comment:
you could send success message along with data? we need to align with this responses format somehow.